### PR TITLE
feat: Added Docker support 🐳 

### PR DESCRIPTION
**Pull request by RatikKapoor**
Containerized development environment using Docker and docker-compose. Instructions have also been updated to match new development flow using Docker 🐳 .

Chanelog:
- Created Dockerfile, docker-compose.yml, and .dockerignore and added containerization functionality
- Updated README to include new development instructions using docker

### CODE REVIEW

**Review by rjb75 — APPROVED** [review]
🐳 is cool and works on windows🖥

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+COPY package.json ./
+COPY yarn.lock ./
+RUN yarn
+# RUN yarn add react-scripts@3.4.1 -g --silent
+
+# add app
+COPY . ./
+
+# start app
+CMD ["yarn", "start"]
```
Comment (lemohammed) [inline Dockerfile:20]: Would it be better to run yarn build here and serving?

Comment (RatikKapoor) [reply Dockerfile:20]: This is for development only right now. I haven't added a production container setup yet. If we run `yarn build` I don't think we'd get hot-reload

Comment (lemohammed) [reply Dockerfile:20]: Ohhh thats truee

**Review by lemohammed — APPROVED** [review]
WOW 🐳

**Pull request merged**